src/Stryker.CLI/Stryker.CLI/CLIOptions.cs

```diff
@@ -52,6 +52,15 @@ public static class CLIOptions
             ArgumentShortName = "-p <projectName>",
             ArgumentDescription = @"Used for matching the project references when finding the project to mutate. Example: ""ExampleProject.csproj""",
             JsonKey = "projectName"
-        };        
+        };
+
+        public static readonly CLIOption<int> MaxConcurrentTestRunners = new CLIOption<int>
+        {
+            ArgumentName = "--maxConcurrentTestRunners",
+            ArgumentShortName = "-m <maxConcurrentTestRunners>",
+            ArgumentDescription = @"Used for running tests with parallel testrunners",
```

Comment:
I don't think the description fully covers what this option does. We could look at how Stryker describes this argument:
``` Specifies the maximum number of concurrent test runners to spawn.
Mutation testing is time consuming. By default Stryker tries to make the most of your CPU, by spawning as many test runners as you have CPU cores.
This setting allows you to override this default behavior.

Reasons you might want to lower this setting:

Your test runner starts a browser (another CPU-intensive process)
You're running on a shared server and/or
Your hard disk cannot handle the I/O of all test runners```

Reply:
Okido, will update the description.